### PR TITLE
implemented a grocery list

### PR DESCRIPTION
**Pull request by nimdolphin**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+body {
+  background-color: orange;
```
Comment (kathrynowy) [inline index.css]: Not made a grocery list
but implemented/created

**Review by kathrynowy — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (nimdolphin) [reply index.css]: fixed

Comment (kathrynowy) [reply index.css]: Where are css variables?

**Review by kathrynowy — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (nimdolphin) [reply index.css]: fixed

**Pull request merged**